setup.py

```diff
@@ -254,6 +254,7 @@ def run(self):
 
 extras["serving"] = deps_list("pydantic", "uvicorn", "fastapi", "starlette")
 extras["speech"] = deps_list("soundfile", "torchaudio")
+extras["tf-flax-speech"] = deps_list("soundfile")
```

Comment:
There is probably a better name than `tf-flax-speech` to mean `soundfile` because the `speech` extras depends on `torchaudio`.

How about adding a single entry `soundfile`? It's unrelated to TF or Flax 

```suggestion
extras["soundfile"] = deps_list("soundfile")
extras["speech"] = deps_list("soundfile", "torchaudio")
```

Reply:
Thought about this as well, but then there will be more dependencies for all torch, flax and tensorflow soon -> how should they be added? For PyTorch we can just add it to `speech` but for TF and Flax, I wouldn't add it to `soundfile`

Reply:
as discussed offline - added `tf-speech` and `flax-speech` now